config-migrate.go

```diff
@@ -40,18 +40,18 @@ func purgeV1() {
 	if err != nil && os.IsNotExist(err) {
 		return
 	}
-	fatalIf(err, "Unable to load config version ‘1’.", nil)
+	fatalIf(err, "Unable to load config version ‘1’.")
 
 	if cv1.Version == "1" {
 		console.Println("Unsupported config version ‘1’ found, removed successfully.")
 		/// Purge old fsUsers.json file
 		configPath, err := getConfigPath()
-		fatalIf(err, "Unable to retrieve config path.", nil)
+		fatalIf(err, "Unable to retrieve config path.")
 
 		configFile := filepath.Join(configPath, "fsUsers.json")
 		os.RemoveAll(configFile)
 	}
-	fatalIf(errors.New(""), "Unexpected version found ‘"+cv1.Version+"’, cannot migrate.", nil)
+	fatalIf(errors.New(""), "Unexpected version found ‘"+cv1.Version+"’, cannot migrate.")
```

Comment:
+	fatalIf(errors.New(""), "Failed to migrate unrecognized config version \""+cv1.Version+"\".")